### PR TITLE
WIP Emscripten build

### PR DESCRIPTION
**Pull request by fckt**

### CODE REVIEW

File: demo/build.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
+set -e
```
Comment (pepyakin) [inline demo/build.sh]: 👍

**Comment by NikVolf** [comment]
Can it be minimized in terms of repository code somehow? We need this builds in 2 more places

**Comment by fckt** [comment]
@NikVolf you mean make it more universal?

**Comment by NikVolf** [comment]
yes, some way to avoid pushing all this code in every repository

**Pull request merged**